plugins/org.jkiss.dbeaver.ext.import_config/src/org/jkiss/dbeaver/ext/import_config/dbvis/DbvisSshServerConfiguration.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.import_config.dbvis;
+
+import org.jkiss.dbeaver.model.net.ssh.SSHConstants;
+
+public class DbvisSshServerConfiguration {
+    private String id;
+    private String name;
+    private String description;
+    private String isEnabled;
+    private String sshHost;
+    private String sshPort;
+    private String sshUserid;
+    private SSHConstants.AuthType authenticationType;
+    private String sshPrivateKeyFile;
+    private String useSshConfigFile;
+    private String sshConfigFile;
+    private String sshKnownHostsFile;
+    private String sshConnectTimeout;
+    private String sshKeepAliveInterval;
+    private String sshMaxAuthTries;
+    private String savePasswords;
+    private String promptPassphrase;
+    private String sshJumpServerId;
+    private String sshImplementationType;
+
+    public String getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
```

Comment:
Maybe making it as recotd will be more suitable for our Java version

Reply:
The class is a kind of POJO encapsulation specific for DBvisualizer products, which, in general, might depend on a particular version of DBvisualizer.